modules/oadp-installing-dpa-1-2-and-earlier.adoc

```diff
@@ -281,37 +281,39 @@ spec:
   configuration:
     velero:
       defaultPlugins:
-        - kubevirt # <1>
-        - gcp # <2>
-        - csi # <3>
-        - openshift # <4>
-      resourceTimeout: 10m # <5>
+        - aws # <1>
```

Comment:
Attach BSL spec has only one backupLocation which is GCP.  AWS plugin is not needed here

Reply:
@PrasadJoshi12 - updated. Thanks